zwave_js_server/model/node.py

```diff
@@ -366,6 +367,9 @@ def handle_metadata_updated(self, event: Event) -> None:
 
     def handle_notification(self, event: Event) -> None:
         """Process a node notification event."""
+        event.data["notification"] = Notification(
```

Comment:
I think we should change the key above when storing the `ValueNotification` to `"value_notification"` to be consistent.

Reply:
@marcelveldt ^ guess I shouldn't have crossed out my comment on the other PR 🙂 

Do we do that as part of this PR or a separate one?

Reply:
I think it's ok to do it here. We can bump and release after this is merged.

Reply:
Go ahead and change here. I'll adjust my PR

Reply:
BTW: I more and more vote for changing the models with prefixes such as ZwaveNode and ZwaveValue and NodeNotification and ValueNotification, see #25

Reply:
Yes, we can do that clean up after this PR if merged.